typed_python/compiler/runtime.py

```diff
@@ -392,6 +392,64 @@ def resultTypeForCall(self, funcObj, argTypes, kwargTypes):
         _resultTypeCache[key] = mergeTypeWrappers(possibleTypes)
         return _resultTypeCache[key]
 
+    @staticmethod
+    def getNativeIRString(typedFunc: Function) -> str:
```

Comment:
so i guess this gets every instantiation of the function? Like - what happens if you call an entrypointed method with two different argument types? reads like you'll get both versions.

Reply:
maybe see `resultTypeForCall` in the same class, which shows how to get a callTarget, which has the exact name of the compiled function based on the argument types